README.md

```diff
@@ -89,23 +90,26 @@ The following table shows how the properties from the Unified Catalog object are
 
 ### Subscription handling
 
-Unified Catalog subscription and Marketplace are different.
-For Unified Catalog there is one subscription per application per Catalog Item
-From the Marketplace it will be translated to one subscription per plan and per application to guarantee that the provider can enforce the quota correct, that is defined in the product plan.
+Unified Catalog subscriptions and Marketplace subscriptions are different.
```

Comment:
There are some core differenced between a Unified Catalog subscription and Marketplace subscription. 
For the Unified Catalog, there is only one subscription allowed per application for a catalog item. In the Marketplace, this will translate to one subscription per product plan, allowing only one application to be registred for a single API resource inside the product, to guarantee that the plan quota can be enforced correctly on the dataplane.